CHANGELOG.md

```diff
@@ -1,3 +1,128 @@
+# Kubermatic 2.17
+
+## [v2.17.0](https://github.com/kubermatic/kubermatic)
+
+### Misc
+
+- New etcd backup and restore controllers [#5668](https://github.com/kubermatic/kubermatic/issues/5668) 
+- Fix kube-system/coredns PodDisruptionBudget matchLabels in user clusters [#6398](https://github.com/kubermatic/kubermatic/issues/6398) 
+- Add `kubermatic-seed` stack target to the Installer [#6435](https://github.com/kubermatic/kubermatic/issues/6435) 
+- add an endpoint to list Vsphere datastores: GET /api/v2/providers/vsphere/datastores [#6442](https://github.com/kubermatic/kubermatic/issues/6442) 
```

Comment:
Please keep a consistent spelling: All changelog items should start with a capital letter. And ideally be in the same time form (e.g. no "Added this, Add this, Adds this", but "Add this, Add that, Add this").

Reply:
Ok

Reply:
Done